vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -111,26 +111,33 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 			var errno syscall.Errno
 			errno = err.(syscall.Errno)
 			msg = fmt.Sprintf("Run '%s' failed: %v (errno=%d).", cmd, err, int(errno))
-
-			// Still below maximum number of retries, log and continue
+			str := C.GoString(ans.errBufP)
+			if str != "" {
+				msg += str
+			}
 			if i < maxRetryCount {
 				log.Warnf(msg + " Retrying...")
 				time.Sleep(time.Second * 1)
 				continue
 			}
-
 			if errno == syscall.ECONNRESET || errno == syscall.ETIMEDOUT {
 				msg += " Cannot communicate with ESX, please refer to the FAQ https://github.com/vmware/docker-volume-vsphere/wiki#faq"
 			}
 		} else {
-			msg = "Internal issue: ret != 0 but errno is not set. Cancelling operation."
+			msg = "Internal issue: ret != 0 but errno is not set. Cancelling operation. "
+			str := C.GoString(ans.errBufP)
```

Comment:
same here - why not simply say 
```golang
msg = "Internal issue: ret != 0 but errno is not set. Cancelling operation. " + C.GoString(ans.errBufP)
```
instead of L127-130 ?